cmake/modules/BuildArrow.cmake

```diff
@@ -49,6 +49,7 @@ function(build_arrow)
     find_package(gRPC REQUIRED)
     find_package(Protobuf REQUIRED)
     find_package(c-ares 1.13.0 QUIET REQUIRED)
+    # find_package(utf8proc 2.2.0 REQUIRED) # BOOST already sets this up
```

Comment:
https://github.com/ceph/ceph/blob/bc11fa7/src/CMakeLists.txt#L893-L899 sets this up just before calling `build_arrow()`. `WITH_SYSTEM_UTF8PROC=OFF` is the default so we build it from submodule

Reply:
Thanks, @cbodley . Still trying to determine when best to put the find_packages at the CMakeLists.txt level and in the BuildArrow level.

And that makes me think that the line in src/CMakeLists.txt should read:
`if(WITH_RADOSGW_SELECT_PARQUET OR WITH_RADOSGW_ARROW_FLIGHT)`

Reply:
So I've added that and am testing the build with `WITH_RADOSGW_SELECT_PARQUET=OFF` to see if everything still works.